### PR TITLE
documentation - new password management policy 

### PR DESCRIPTION
**Pull request by marwanehcine**
Steps to enable rotation password management policy

### CODE REVIEW

File: docs/setup/openldap.md [hunk 1]
```diff
@@ -132,3 +132,27 @@ ldapvi --host localhost -D "cn=admin,dc=georchestra,dc=org" -w "secret" -b "dc=g
 
  * [Apache Directory Studio](http://directory.apache.org/studio/), a powerful desktop client.
  * our own [console](/console/README.md) web application, available at ```/console/manager/``` to  members of the ```SUPERUSER``` group, is probably the easiest one.
+
+# Enabling rotation policy for passwords management 
+
+To enable rotation policy for passwords management, please run the following commands:
+
+sudo ldapadd -Y EXTERNAL -H ldapi:/// -f ppolicy-rotation.ldif
```
Comment (fvanderbiest) [inline docs/setup/openldap.md:140]: Please use triple quotes, as above, for commands.

File: docs/setup/openldap.md [hunk 2]
```diff
@@ -132,3 +132,27 @@ ldapvi --host localhost -D "cn=admin,dc=georchestra,dc=org" -w "secret" -b "dc=g
...
+
+sudo ldapadd -Y EXTERNAL -H ldapi:/// -f rotationpolicyoverlay.ldif
+
+To disable password expire for no humain users (geoserver_privileged_user, idatafeeder), please run the following commands::
```
Comment (fvanderbiest) [inline docs/setup/openldap.md:144]: "To disable password expiration for internal users"
Also an extra `:` char at EOL

File: docs/setup/openldap.md [hunk 3]
```diff
@@ -132,3 +132,27 @@ ldapvi --host localhost -D "cn=admin,dc=georchestra,dc=org" -w "secret" -b "dc=g
...
+
+sudo ldapadd -Y EXTERNAL -H ldapi:/// -f pwd_no_expire.ldif
+
+sudo ldapadd -Y EXTERNAL -H ldapi:/// -f pwd_no_expire_users.ldif
+
+If  rotation policy for passwords management is enabled, password has to be set after 12 months.
+Alerts will be shown to user during last month.
+
```
Comment (fvanderbiest) [inline docs/setup/openldap.md:152]: extra blank lines to be removed

File: docs/setup/openldap.md [hunk 4]
```diff
@@ -132,3 +132,27 @@ ldapvi --host localhost -D "cn=admin,dc=georchestra,dc=org" -w "secret" -b "dc=g
...
+sudo ldapadd -Y EXTERNAL -H ldapi:/// -f rotationpolicyoverlay.ldif
+
+To disable password expire for no humain users (geoserver_privileged_user, idatafeeder), please run the following commands::
+
+sudo ldapadd -Y EXTERNAL -H ldapi:/// -f pwd_no_expire.ldif
+
+sudo ldapadd -Y EXTERNAL -H ldapi:/// -f pwd_no_expire_users.ldif
+
+If  rotation policy for passwords management is enabled, password has to be set after 12 months.
```
Comment (fvanderbiest) [inline docs/setup/openldap.md:150]: This duration can be set with the XXX option in the XXX file.

**Review by fvanderbiest — CHANGES_REQUESTED** [review]
Thanks for the documentation.
Minor comments to be addressed.

**Review by pmauduit — APPROVED** [review]

**Pull request merged**